### ISSUE TITLE
Reading an existing .json file written out using TinyDB

### ISSUE THREAD

**Issue by tomwallis** [msiemens/tinydb#67]
Apologies if this is a stupid question, but I have not seen the answer in your docs.

A colleague has passed me a `.json` file written out using TinyDB. I would like to read its contents. However, if I use

```
db = TinyDB('my_file.json')
```

then `db` is empty, and the file on disk is overwritten (contains one line `{"_default": {}}`). What's the correct way to read this file into a Python interpreter? I see no "load" method in the docs and `tinydb.Storage.read` doesn't do this.

**Comment by eugene-eeo** [comment]
Slightly weird question but is the JSON file valid? The last time I worked on the code we used a `try..except` block to test if files were empty, so if the file was manually modified and some minor issues were present (think missing quotes, unbalanced parens etc) then it would result in the same case as an empty file. So try loading it manually using `json.loads` to see if it works.

**Comment by tomwallis** [comment]
Ah, you seem to have it right. 

```import json
with open('tom_mean_feature_activations.json', 'r') as f:
    data = json.load(f)

...


/Users/user/miniconda3/envs/default/lib/python3.4/json/decoder.py in raw_decode(self, s, idx)
    357         """
    358         try:
--> 359             obj, end = self.scan_once(s, idx)
    360         except StopIteration as err:
    361             raise ValueError(errmsg("Expecting value", s, err.value)) from None

ValueError: Expecting ',' delimiter: line 1 column 12582912 (char 12582911)

```

The `.json` file my colleague gave me appears to be invalid. Issue solved, but I suppose it would be helpful if tinyDB reported an error rather than silently overwriting the file.

**Comment by msiemens** [comment]
Actually I'd like to keep this open. TinyDB currently assumes that if the JSON module raises a `ValueError`, the database file was empty. But as in this case, that's a pretty shaky assumption and something I want to revisit for a potential v3.0.

**Comment by eugene-eeo** [comment]
Unhelpful but encouraging :+1: for looking into this issue.

**Comment by msiemens** [comment]
This should be closed in the 3.0 release, I forgot to close this issue.
